src/SearchResults/index.js

```diff
@@ -133,7 +133,7 @@ var SearchResults = function( state, algoliaResponse ) {
     forEach( result.facets, function( facetResults, dfacet ) {
       var position = disjunctiveFacetsIndices[ dfacet ];
 
-      var dataFromMainRequest = mainSubResponse.facets[ dfacet ];
+      var dataFromMainRequest = mainSubResponse.facets && mainSubResponse.facets[ dfacet ] || {};
```

Comment:
Parenthesis would be nice here.
Also, isn't `mainSubResponse.facets[ dfacet ]` is an array, so shouldn't it be `[]`?